queries.sql

```diff
@@ -58,4 +58,16 @@ SELECT animals.name AS animal_with_most_visits, COUNT(visits.animals_id) AS Anim
 SELECT animals.name AS first_animal_to_visit_Maisy, vets.name, MIN(visits.date_of_visit) FROM VISITS JOIN animals ON animals.id = visits.animals_id JOIN vets ON vets.id = visits.vets_id WHERE vets.name = 'Maisy Smith' GROUP BY animals.name, vets.name, visits.date_of_visit LIMIT 1;
 SELECT * FROM visits JOIN animals ON visits.animals_id = animals.id JOIN vets ON vets.id = visits.vets_id ORDER BY visits.date_of_visit DESC LIMIT 1;
 SELECT COUNT(*) AS visits_with_a_vet_tha_did_no_specialize FROM visits JOIN (SELECT vets.id FROM vets FULL JOIN specializations ON vets.id = specializations.vets_id FULL JOIN species ON species.id = specializations.species_id WHERE specializations.species_id IS NULL) AS vet ON vet.id = visits.vets_id;
-SELECT s.name AS species_name_Maisy_should_consider, COUNT(vis.date_of_visit) AS total_visits FROM visits AS vis JOIN animals AS a ON vis.animals_id = a.id JOIN species AS s ON a.species_id = s.id JOIN vets AS v ON vis.vets_id = v.id WHERE v.name =  'Maisy Smith' GROUP BY s.name ORDER BY total_visits DESC LIMIT 1;
\ No newline at end of file
+SELECT s.name AS species_name_Maisy_should_consider, COUNT(vis.date_of_visit) AS total_visits FROM visits AS vis JOIN animals AS a ON vis.animals_id = a.id JOIN species AS s ON a.species_id = s.id JOIN vets AS v ON vis.vets_id = v.id WHERE v.name =  'Maisy Smith' GROUP BY s.name ORDER BY total_visits DESC LIMIT 1;
+
+
+-- databasee performance
+EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animals_id = 4;
+
+EXPLAIN ANALYZE SELECT * FROM visits where vets_id = 2;
```

Comment:
- [ ] It would be nice if the before screenshot should have an execution time should be around 1 sec (1000 ms) but I have seen that the first and third before screenshots are having execution times of less than 1000ms 😨 Kindly populate your database with some data to make your before execution time more than 1000ms 👍